pingcap/tidb
# metrics: distinguish transactions by mode

Signed-off-by: Shuaipeng Yu <jackysp@gmail.com>



### What problem does this PR solve?

Issue Number: close https://github.com/pingcap/tidb/issues/17377 

Problem Summary:

### What is changed and how it works?

Proposal: [xxx](url) 

What's Changed:
Use "pessimistic" and "optimistic" to distinguish transactions.

How it Works:

### Related changes

- Need to cherry-pick to the release branch

### Check List 

Tests 

- Manual test (add detailed scripts or steps below)
* start a new cluster with this branch
* check out the metrics

### Release note 
* No release note
-
